plugins/ltac/tactic_debug.ml

```diff
@@ -54,24 +79,24 @@ let db_pr_goal gl =
 let db_pr_goal =
   Proofview.Goal.enter begin fun gl ->
   let pg = db_pr_goal gl in
-  Proofview.tclLIFT (msg_tac_notice (str "Goal:" ++ fnl () ++ pg))
+  Proofview.tclLIFT (Comm.goal pg)
```

Comment:
I would keep printing "Goal:\n" as a separate output before the Comm.goal for the moment.

Reply:
Sounds good, will amend.

Reply:
After a second review I think this should be done client-side.